remove-or-recrawl-urls.html

```diff
@@ -24,44 +24,35 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     <title>Simple form for removing URLs using a feed</title>
   <script type="text/javascript">
-  //<![CDATA[
-  function buildFeed(header, records, footer) {
+  function buildFeed(header, records, footer, attributes ) {
     var records_xml = '';
     var rl = records.length;
+    attributes = typeof attributes !== 'undefined' ?  attributes : ''; 
     for (r = 0; r < rl; r++) {
-      if (records[r].length != 0) {
-        records_xml += '    <record url="' + records[r] + '" />\n';
-      }
+        if (records[r].length != 0) {
```

Comment:
There was different indentation in different parts of the JS script. I aligned it to be the same across the script.